### DISCUSSION TITLE
Including "On this page" in about.qmd

### DISCUSSION THREAD

**Discussion by DidDrog11** [quarto-dev/quarto-cli#3099] [Q&A]
Hi,

Is there a way to use sections on the about.qmd page? I have it available elsewhere in the website.

**Comment by mcanouil** [comment]
What is exactly your use case / need?

One way would be to use a simple page and design it instead of using one of the "about" templates.

**Reply by DidDrog11** [reply]
Only that my About page has several headings covering different roles. It could make it easier to navigate with the "On this page" section but yes I could try and use a standard page and use the code from the About templates rather than the other way around.

Thanks.

**Reply by mcanouil** [reply]
Or formerly make a Feature Request for a TOC in "about" templates.
